http/src/main/resources/init-resources/startup.sh

```diff
@@ -262,29 +269,10 @@ END
     if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
         echo "Restarting Rstudio Container $GOOGLE_PROJECT / $CLUSTER_NAME..."
 
-        # Make sure when runtimes restarts, they'll get a new version of rstudio docker compose file
-        $GSUTIL_CMD cp gs://${INIT_BUCKET_NAME}/`basename ${RSTUDIO_DOCKER_COMPOSE}` $RSTUDIO_DOCKER_COMPOSE
-
-tee /var/variables.env << END
-WORK_DIRECTORY=${WORK_DIRECTORY}
-RSTUDIO_SERVER_NAME=${RSTUDIO_SERVER_NAME}
-RSTUDIO_DOCKER_IMAGE=${RSTUDIO_DOCKER_IMAGE}
-RSTUDIO_USER_HOME=${RSTUDIO_USER_HOME}
-GOOGLE_PROJECT=${GOOGLE_PROJECT}
-RUNTIME_NAME=${RUNTIME_NAME}
-OWNER_EMAIL=${OWNER_EMAIL}
-PET_SA_EMAIL=${PET_SA_EMAIL}
-WELDER_ENABLED=${WELDER_ENABLED}
-MEM_LIMIT=${MEM_LIMIT}
-SHM_SIZE=${SHM_SIZE}
```

Comment:
This is a tricky one, because users can update their VMs to ask for more memory, we were updating the docker compose to modify the docker shared memory accordingly, which is why we modified the script to get the new env variable and do a docker compose up 🤔